--- msn-transport-cvs/configure.in	2003-09-22 08:04:47.000000000 +1000
+++ msn-transport-new/configure.in	2003-09-22 08:05:32.000000000 +1000
@@ -1,7 +1,7 @@
 AC_INIT(src/init.c)
 
 AM_CONFIG_HEADER(include/config.h)
-AM_INIT_AUTOMAKE(msn-transport,1.2.8pre4)
+AM_INIT_AUTOMAKE(msn-transport,1.2.8pre5)
 
 AC_PROG_CC
 AM_SANITY_CHECK
@@ -50,6 +50,7 @@
 AC_MSG_CHECKING(for curl.h)
 if test -f "$CURL_PATH/curl.h"; then
     AC_MSG_RESULT(Found);
+    LIBS="$LIBS -lcurl"
 else
     AC_MSG_RESULT(Error);
     AC_MSG_ERROR([
